src/backend/core/tests/api/test_entitlements_messages_admin.py

```
@@ -1148,3 +1157,159 @@ def test_operator_admin_passthrough_email_must_match_exactly():
+    assert entitlements["can_access"] is False
+    # The (non-cross-org) admin resolver did not run: no admin fields leak in.
+    assert "is_admin" not in entitlements
+    assert "can_admin_maildomains" not in entitlements
+
+
+def test_messages_admin_resolver_runs_without_active_subscription_flag():
+    """The Messages admin resolver is flagged to run without an active
+    subscription; the base and extended resolvers are not."""
+    from core.entitlements.resolvers import (  # pylint: disable=import-outside-toplevel
+        get_admin_entitlement_resolver,
+    )
```

Ruff (0.15.21)

[warning] `import` should be at the top-level of a file

(PLC0415)